### PR TITLE
add it.cuteworks.pacmanlogviewer

### PR DESCRIPTION
**Pull request by gcala**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: it.cuteworks.pacmanlogviewer.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "plv",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--filesystem=/var/log/pacman.log:ro"
```
Comment (hfiguiere) [inline it.cuteworks.pacmanlogviewer.json:13]: that can't be a file.

Comment (gcala) [reply it.cuteworks.pacmanlogviewer.json:13]: not even the log file of the package manager?

**Comment by gcala** [comment]
removed file

**Comment by barthalion** [comment]
bot, build it.cuteworks.pacmanlogviewer

**Comment by hfiguiere** [comment]
bot, build it.cuteworks.pacmanlogviewer

**Review by hfiguiere — CHANGES_REQUESTED** [review]
it fail to build.

**Comment by hfiguiere** [comment]
you can also trigger builds

bot, build it.cuteworks.pacmanlogviewer

**Comment by hfiguiere** [comment]
I don't have that kind of power. Arm yourself with some patience.

**Comment by hfiguiere** [comment]
also did you test it?

**Comment by gcala** [comment]
I sussessfully tested it. Sorry for asking the merge, I thought you were an "admin". Is the publication automatic? Thanks

**Comment by barthalion** [comment]
I don't really understand how it's supposed to work? The app doesn't have direct filesystem access to /var/log, and doesn't offer a file dialog to choose pacman.log.

**Comment by gcala** [comment]
> I don't really understand how it's supposed to work? The app doesn't have direct filesystem access to /var/log, and doesn't offer a file dialog to choose pacman.log.

Was asked to remove the file: https://github.com/flathub/flathub/pull/3937#discussion_r1118096793

Application has a selection file dialog (icon on upper right corner)

**Comment by hfiguiere** [comment]
no. I said it will not work specifying a file.

**Comment by gcala** [comment]
> no. I said it will not work specifying a file.

I tested a local flatpak and it worked. So can I restore the line?

**Comment by gcala** [comment]
bot, build it.cuteworks.pacmanlogviewer

**Comment by gcala** [comment]
bot, build it.cuteworks.pacmanlogviewer

**Comment by gcala** [comment]
@barthalion I have restored access to `/var/log/pacman.log`. Build was successfull and testing too. Do you think is it ready for official build? Thanks.

**Comment by gcala** [comment]
bot, build it.cuteworks.pacmanlogviewer

**Comment by hfiguiere** [comment]
why do you do that? The PR is closed.

**Pull request closed without merging**